bundles/org.openhab.binding.siemenshvac/README.md

```diff
@@ -0,0 +1,111 @@
+# SiemensHVAC Binding
+
+This binding provides support for the Siemens HVAC controller ecosystem, and the Web Gateway interface OZW672.
+A typical system is composed of:
+         
+![Diagram](doc/Diagram.png)                 
+ 
+There's a lot of different HVAC controllers depending on model in lot of different PAC constructors.
+Siemens RVS41.813/327 inside a Atlantic Hybrid Duo was used for the development, and is fully supported and tested.
+
+Siemens have a complete set of controller references under the name "Siemens Albatros".
+Here is a picture of such device.
+You can also find this device in other types of heating systems: boiler or solar based.
+
+![](doc/Albatros.jpg)
+
+You will find some information about the OZW672.01 gateway on the Siemens web site: 
+
+[OZW 672 Page](https://hit.sbt.siemens.com/RWD/app.aspx?rc=FR&lang=fr&module=Catalog&action=ShowProduct&key=BPZ:OZW672.01)
+
+With this binding, you will be able to:
+
+- Consult the different parameters of your system, like temperature, current heating mode, water temperature, and many more.
+- Modify the functioning mode of your device: temperature set point, heating mode, and others.
+
+## Supported Things
+
+Support many different things as the thing type is handle by autodiscovery.
+
+Mainly, it will first discover the gateway.
+Currently test and support is the OZW672.x series.
+No test done with OZW772.x series, but it should work as well.
```

Comment:
I removed this test

Reply:
What did you remove? The discovery still checks explicitly for OZW672, so maybe it should be stated that it might work, but that the Bridge will need to be created manually in this case?